packages/vue-components/src/Searchbar.vue

```diff
@@ -4,6 +4,7 @@
     <input
       v-else
       v-model="value"
+      v-bind:style="{width: placeholder.length + 'ch'}"
```

Comment:
Try running `npm run lint` and you should see this warning:
>  7:7  warning  Unexpected 'v-bind' before ':'  vue/v-bind-style

If you have ESlint installed you should be able to see a warning and how to resolve it, if you are using VSCode:
![image](https://user-images.githubusercontent.com/41845017/177187935-ec56566c-93af-419b-80a0-de69a3209a51.png)